src/BloomBrowserUI/bookEdit/toolbox/music/musicToolControls.tsx

```diff
@@ -171,6 +178,10 @@ export class MusicToolControls extends React.Component<{}, IMusicState> {
         player.setAttribute("src", musicUrl + "?nocache=" + new Date().getTime());
         player.volume = this.getPlayerVolumeFromAttributeOnPage(audioFileName);
         player.play();
+        // Play the audio during animation
+        this.narrationPlayer = new AudioRecording();
+        this.narrationPlayer.setupForListen();
```

Comment:
I first was thinking... "is it supposed to listen for some event"? I get that it is "Listen" because that's the button the user plays, but... "Play" would sure be clearer in the code :-)